libs/service-portal/documents/src/components/DocumentLine/DocumentLine.tsx

```diff
@@ -24,6 +25,7 @@ interface Props {
 const DocumentLine: FC<Props> = ({ documentLine, userInfo, img }) => {
   const { formatMessage } = useLocale()
   const onClickHandler = () => {
+    getAccessToken()
```

Comment:
Unfortunately you need to await and use this access token. When the token is not expired it takes 0ms, but may take up to a second if it's expired.

We also need to test this in the main browsers and see if this async delay triggers a popup blocker. :(

Reply:
Ah I see, too good to be true then 😆